src/components/elements/text-body/text-body.tsx

```diff
@@ -0,0 +1,59 @@
+import { Text } from "@lifesg/react-design-system/text";
+import { TestHelper } from "../../../utils";
+import { IGenericFieldProps } from "../../frontend-engine";
+import { ITextbodySchema } from "./types";
+
+export const TextBody = (props: IGenericFieldProps<ITextbodySchema>) => {
+	// =============================================================================
+	// CONST, STATE, REF
+	// =============================================================================
+	const {
+		id,
+		schema: { children, ...otherSchema },
+	} = props;
+
+	// =============================================================================
+	// HELPER FUNCTIONS
+	// =============================================================================
+	const getTestId = (id: string): string => {
+		return TestHelper.generateId(id, "textbody");
+	};
+
+	const hasNestedFields = (): boolean => {
+		return typeof children !== "string";
+	};
+
+	// =============================================================================
+	// RENDER FUNCTIONS
+	// =============================================================================
+	const renderTextBody = (): JSX.Element[] | string[] | string => {
```

Comment:
is it possible to parse HTML from string? we will want to serve HTML text from JSON